rental_manager.rb

```diff
@@ -0,0 +1,61 @@
+require_relative 'rental'
+
+class RentalManager
+  attr_reader :rentals
+
+  def initialize(books, people)
+    @books = books
+    @people = people
+    @rentals = []
+  end
+
+  def add_rental
+    puts ''
+    puts 'Select a book from below by number: '
+
+    @books.each_with_index { |book, index| puts "(#{index}) Title: #{book.title}, Author: #{book.author}" }
+    puts ''
+    book_num = gets.chomp.to_i
+    puts ''
+
+    puts 'Select a person from below by number: '
+    @people.each_with_index do |person, index|
+      puts "(#{index}) [#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+    puts ''
+    person_num = gets.chomp.to_i
+    puts ''
+
+    print 'Date(yyyy/mm/dd): '
+    date = gets.chomp
```

Comment:
Just as I have requested previously kindly take every method for requesting and validating input to the main.rb file to fulfill the single responsibility rule👍.